deepchem/molnet/load_function/bbbp_datasets.py

```diff
@@ -47,19 +50,22 @@ def load_bbbp(featurizer='ECFP', split='random', reload=True):
       deepchem.trans.BalancingTransformer(transform_w=True, dataset=dataset)
   ]
 
-  print("About to transform data")
+  logger.info("About to transform data")
   for transformer in transformers:
     dataset = transformer.transform(dataset)
 
-  splitters = {
-      'index': deepchem.splits.IndexSplitter(),
-      'random': deepchem.splits.RandomSplitter(),
-      'scaffold': deepchem.splits.ScaffoldSplitter()
-  }
-  splitter = splitters[split]
-  train, valid, test = splitter.train_valid_test_split(dataset)
+  if split == None:
+    return bbbp_tasks, (dataset, None, None), transformers
+  else:
```

Comment:
this else is unnecessary and adds an extra layer of indentation.  Please remove.

Reply:
Here and elsewhere.  As a general rule of thumb extra layers of indentation are bad.  It makes the code harder to read and makes the programmer store state in the if/else tree in their mind.